OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/IUserManager.kt

```diff
@@ -25,6 +26,16 @@ interface IUserManager {
      */
     val pushSubscription: IPushSubscription
 
+    /**
+     * onesignalId, empty if this is current unavailable
+     */
```

Comment:
Just a few more suggestions. Since we are documenting this getter, let's tell them what this ID is:

> "The UUID generated by OneSignal to represent a user, empty if this is currently unavailable" 


(I pulled from [our docs](https://documentation.onesignal.com/docs/aliases-external-id#onesignal-id))

Reply:
Good point! I have modified all the comments to make them clear and consistent to our docs.